addons/rookupgrade/10to14/install.sh

```diff
@@ -251,6 +278,11 @@ function rookupgrade_10to14_upgrade() {
         kubectl -n rook-ceph set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v1.4.9
         kubectl apply -f "$upgrade_files_path/rook-ceph-tools-14.yaml"
 
+        log "Awaiting up to 5 minutes to check Rook Ceph Pod(s) are Running"
+        if ! spinner_until 300 check_for_running_pods "rook-ceph"; then
```

Comment:
I don't understand what check_for_running_pods does that `"$DIR"/bin/kurl rook wait-for-rook-version` does not do. If there is some gap in functionality is it possible to add that to the wait-for-rook-version command rather than add more bash?

Reply:
same above : https://github.com/replicatedhq/kURL/pull/4312#discussion_r1195435294